### PR TITLE
Remove link text from Dashboard AccountList

### PR DESCRIPTION
**Pull request by vilsbole**
Following all hands design conversation:
Remove link text from table header.

### Before
Screenshot 2020-04-10 at 11 06 57 AM

### After
Screenshot 2020-04-10 at 11 12 19 AM

### CODE REVIEW

File: common/v2/components/DashboardPanel.tsx [hunk 1]
```diff
@@ -79,20 +79,39 @@ export const DashboardPanel = ({
...
+            <img src={settingsIcon} alt={'settings'} width={24} />
+            <Typography>{headingRight}</Typography>
+          </SRouterLink>
+        </>
+      );
+    } else if (headingRight && !actionLink) {
+      return <Typography>{headingRight}</Typography>;
+    } else if (!headingRight && actionLink) {
+      return (
+        <>
```
Comment (blurpesec) [inline common/v2/components/DashboardPanel.tsx:96]: is this fragment needed?

**Comment by LuckyLuker** [comment]
It looks like the size of the gear icon shrank when the text was removed

File: common/v2/components/DashboardPanel.tsx (hunk 1 above)
Comment (vilsbole) [reply common/v2/components/DashboardPanel.tsx:96]: Indeed. Removing.

**Comment by vilsbole** [comment]
> It looks like the size of the gear icon shrank when the text was removed

@LuckyLuker Yes it's an effort to make the link less prominent. I can reverse if you prefer?

**Comment by vilsbole** [comment]
Update icon size to match 30x30 from settings page.

**Comment by LuckyLuker** [comment]
Looks like this was already resolved. Thanks, Emmanuel!

**Pull request merged**